File: app/src/main/java/com/artmurka/artmurkaapp/Model/Pojo/ItemList/ItemBasket/EntryWeight.java

package com.artmurka.artmurkaapp.Model.Pojo.ItemList.ItemBasket;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class EntryWeight {

    @SerializedName("weight")
    @Expose
    private int weight;
    @SerializedName("weight_raw")
    @Expose
    private int weightRaw;

    public int getWeight() {
        return weight;
    }

    public void setWeight(int weight) {
        this.weight = weight;
    }

    public int getWeightRaw() {
        return weightRaw;
    }

    public void setWeightRaw(int weightRaw) {
        this.weightRaw = weightRaw;
    }

}

File: app/src/main/java/com/artmurka/artmurkaapp/data/model/pojo/itemlist/itembasket/EntryWeight.kt
package com.artmurka.artmurkaapp.data.model.pojo.itemlist.itembasket

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class EntryWeight {

    @SerializedName("weight")
    @Expose
    var weight: Int = 0
    @SerializedName("weight_raw")
    @Expose
    var weightRaw: Int = 0

}
